### PR TITLE
Minify svg exports using svgo

### LINKED ISSUE #366: Minify SVGs?
On every page, before the interactive charts are loaded, we first load the SVGs for the charts. 

(Try search for "svg" on page load on any entry, e.g. https://ourworldindata.org/coronavirus-data – the total is 724KB gzipped)

We could _probably_ minify our SVGs significantly without losing any detail. I've looked at [SVGO](https://github.com/svg/svgo/blob/master/README.md) for this, though there may be other options.

### PR DESCRIPTION
**Pull request by MarcelGerber**
Resolves #366.

This adds [svgo](https://www.npmjs.com/package/svgo), a svg minifier, to the baking step and automatically reduces the size of all chart previews.

### CODE REVIEW

File: site/server/svgPngExport.tsx [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
 declare var global: any
 global.window = { location: { search: "" } }
 global.App = { isEditor: false }
 
 import { ChartConfig, ChartConfigProps } from "charts/ChartConfig"
 
+const svgoConfig: svgo.Options = {
+    floatPrecision: 1,
+    plugins: [{ collapseGroups: false }, { removeViewBox: false }]
+}
```
Comment (MarcelGerber) [inline site/server/svgPngExport.tsx]: This config is maybe up to some debate, and its worth noting that it extends the [default config](https://github.com/svg/svgo/blob/master/.svgo.yml).

**Comment by MarcelGerber** [comment]
For comparison:
* Currently, [PM25-air-pollution.svg](https://ourworldindata.org/grapher/exports/PM25-air-pollution.svg) has a size of 155KB, or 51KB gzipped
* The [new one](https://icedwave-dl.s3.eu-central-1.amazonaws.com/PM25-air-pollution.svg) has a size of 85KB, or 28KB gzipped

You can compare these two for any visible visual differences.

**Comment by breck7** [comment]
(note: I moved this comment from the issue to the PR)

Awesome @MarcelGerber !

The code LGTM.

A quick cd owid-static/grapher/exports; du -hc *.svg shows me that the current static SVGs sum to 264MB.

I will merge this now and report back the global effect. I'll also do a broader visual inspection before pushing.

**Comment by MarcelGerber** [comment]
Please don't enable this just yet.
I've just noticed that the link in the footer is stripped of its href, so e.g. OurWorldInData.org/air-pollution/ cannot be clicked.
I will look for a setting to change this behavior later.

**Comment by breck7** [comment]
@MarcelGerber I put it behind a flag for now in e60bc15b41f7cac8e07fe929e1d3317f4b986be9

**Comment by MarcelGerber** [comment]
Yep I saw that, just put it out there so you don't miss it and accidentally enable it.

**Pull request merged**